ext/zend_test/tests/gh8575.phpt

```diff
@@ -0,0 +1,13 @@
+--TEST--
+CLI: stderr is available in mshutdown
+--SKIPIF--
+<?php if (php_sapi_name() != "cli")) die('skip: cli test only'); ?>
```

Comment:
Nit: if a test is skipped, the message just removes the "skip", but not a trainling colon, so in this case it would be "reason: : cli test only". Thus:
```suggestion
<?php if (php_sapi_name() != "cli")) die('skip cli test only'); ?>
```

Reply:
🤦🏻 and thanks!